cmake/SearchForStuff.cmake

```diff
@@ -679,13 +703,18 @@ endif ()
 
 ########################################
 # Find gdal
-include (FindGDAL)
+find_package(GDAL CONFIG)
+if ("${GDAL_LIBRARY}" STREQUAL "")
+  unset(GDAL_FOUND CACHE)
+  include (FindGDAL)
+endif()
 if (NOT GDAL_FOUND)
   message (STATUS "Looking for libgdal - not found")
   BUILD_WARNING ("GDAL not found, Digital elevation terrains support will be disabled.")
   set (HAVE_GDAL OFF CACHE BOOL "HAVE GDAL" FORCE)
 else ()
   message (STATUS "Looking for libgdal - found")
+  message (STATUS "GDAL_LIBRARY: ${GDAL_LIBRARY}")  
```

Comment:
Debug message, perhaps we can remove it?

Reply:
done.